test/per_file_coverage.sh

```diff
@@ -25,7 +25,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/singleton:95.7"
 "source/common/tcp:94.6"
 "source/common/thread:0.0" # Death tests don't report LCOV
-"source/common/tracing:96.1"
+"source/common/tracing:95.3"
```

Comment:
What's the cause of this being lower? Can you add more tests?

Reply:
This fix came from this change. https://github.com/envoyproxy/envoy/pull/19030/files#diff-b34e5ba9bf6eb09787a43f6a1f8ccead0e9e7f412eaa12c2b7d06f7d4d150ba2R36
We are using `TracerFactoryContext::transportSocketFactoryContext()` only [here](https://github.com/envoyproxy/envoy/pull/19030/files#diff-dfccda64c35905772b119e9525441cb73ff37a85353e8d0dac4b0dfb960e848fR76-R77). But, we are using `TracerFactoryContextMock` in the tracing test. So we can't cover those lines in the current implementation.

Reply:
Can you add an integration test to cover this?